packages/cdktf-cli/templates/python-pip/.hooks.sscaff.js

```diff
@@ -41,16 +41,15 @@ exports.post = options => {
 function terraformCloudConfig(baseName, organizationName, workspaceName) {
   template = readFileSync('./main.py', 'utf-8');
 
-  const result = template.replace(`MyStack(app, "${baseName}")`, `stack = MyStack(app, "${baseName}")
-stack.add_override('terraform.backend', {
-  'remote': {
-    'hostname': 'app.terraform.io',
-    'organization': '${organizationName}',
-    'workspaces': {
-      'name': '${workspaceName}'
-    }
-  }
-})`);
+  const templateWithImports = template.replace(`from cdktf import App, TerraformStack`,
+    `from cdktf import App, TerraformStack, RemoteBackend, NamedRemoteWorkspace`)
+
+  const result = templateWithImports.replace(`MyStack(app, "${baseName}")`, `stack = MyStack(app, "${baseName}")
+RemoteBackend(stack,
```

Comment:
I think we'd need to import the `RemoteBackend` class somehow. Since this is conditionally, we can't just add it over [here](https://github.com/hashicorp/terraform-cdk/blob/4974545cf8fa3f08198e3cac99db218e249985bf/packages/cdktf-cli/templates/python-pip/main.py#L3) but would have to dynamically inject it as well. Or am I missing something here?

Reply:
Oh, sorry - Just realized you did this a few lines above :D